src/blocks/weather/met_no.rs

```diff
@@ -160,8 +178,8 @@ impl WeatherProvider for Service<'_> {
             .error("No location given")?;
 
         let querystr: HashMap<&str, String> = map! {
-            "lat" => lat,
-            "lon" => lon,
+            "lat" => &lat,
```

Comment:
https://api.met.no/weatherapi/sunrise/3.0/documentation

>* **offset** (optional), timezone offset, on the format +HH:MM or -HH:MM
[Timezone offset](https://api.met.no/weatherapi/sunrise/3.0/documentation#___top)
The optional offset parameter is the difference between local time and UTC, as defined in ISO 8601. This is used mainly for returning timestamps in local time, but is also used in some instances to calculate the correct date period. If omitted, all times will be given in UTC.
>
>Presently timezones for the offset parameter range from -12:00 to +15:00, although this may change in the future. For GMT, only +00:00 is allowed. You can see a list of all current timezones on https://en.wikipedia.org/wiki/List_of_UTC_time_offsets.

Reply:
I am not sure if this is necessary. After some testing, it seems like the status bar correctly renders the time in the correct timezone out of the box. It makes more sense for me to receive the sunset and sunrise times in GMT when it will be adjusted to the correct timezone when displayed anyways. What do you think, is it OK as it is?

![image](https://github.com/greshake/i3status-rust/assets/20906912/efbc5a91-c88c-4fa0-96d1-8df307a9cf04)

![image](https://github.com/greshake/i3status-rust/assets/20906912/42c646a7-970a-4728-84ae-e1f6c85c8b7e)

(sorry late reply, been very busy lately)

Reply:
No worries. It looks like when the date is omitted I am getting the sunrise/sunset for tomorrow instead of today. You're right that it doesn't seem to matter if the offset is included. 

The example I was using is https://api.met.no/weatherapi/sunrise/3.0/sun?lat=40.43&lon=-74.01&date=2024-05-13 (on 2024-05-13) I see the times for 2024-05-14. I think that the default date is the date in GMT